### PR TITLE
Resolving the span Tag bug and Make changes in Search Patient ui

### PR DESCRIPTION
**Pull request by sandipbhuyan**
**BEFORE**
screen shot 2018-02-21 at 11 15 28 am
**AFTER**
screen shot 2018-02-21 at 11 15 44 am

### CODE REVIEW

**Comment by sandipbhuyan** [comment]
Nope @aethelwulffe

File: interface/reports/non_reported.php [hunk 1]
```diff
@@ -390,7 +390,7 @@ function tr($a)
             '>
             <span>
               <?php xl('Refresh','e'); ?>
-            </spain>
+            </span>
```
Comment (teryhill) [inline interface/reports/non_reported.php]: This is/was in another PR

File: interface/patient_tracker/patient_tracker.php [hunk 2]
```diff
@@ -768,6 +768,7 @@ function openNewTopWindow(newpid,newencounterid) {
         $.datetimepicker.setLocale('<?= $DateLocale;?>');
     });
     $("#form_apptstatus").addClass('form-control input-sm');
+    $('#form_apptstatus').removeAttr('style');
```
Comment (teryhill) [inline interface/patient_tracker/patient_tracker.php]: This is/was in another PR

Comment (aethelwulffe) [reply interface/patient_tracker/patient_tracker.php]: They were, but there are no conflicts as such.  They will mash together.

Comment (aethelwulffe) [reply interface/patient_tracker/patient_tracker.php]: He might not have been able to test with broken code...you know?

File: interface/reports/non_reported.php (hunk 1 above)
Comment (sandipbhuyan) [reply interface/reports/non_reported.php]: this is another span tag bug that i Have resolved . I missed this one on the first time.

**Comment by aethelwulffe** [comment]
It looks right to me.

**Comment by sandipbhuyan** [comment]
@aethelwulffe  Abort #1015 PR. Merge this one.

**Comment by aethelwulffe** [comment]
OK, I am going to close #1015

**Comment by aethelwulffe** [comment]
Closed #1015

**Comment by aethelwulffe** [comment]
bump

**Pull request merged**